schema/encoding/jsonschema/all_go16_test.go

```diff
@@ -0,0 +1,11 @@
+// +build !go1.7
+
+package jsonschema_test
```

Comment:
I was hoping to link to a blog post or some docs explaining the "_test" suffix, but I couldn't find it just now. However, it's a very common pattern, and you can also see it's commonly used in the Go standard library. E.g. https://golang.org/src/strings/reader_test.go